apps/explorer/src/app/routes/parties/id/index.tsx

```diff
@@ -59,11 +61,16 @@ const PARTY_ASSETS_QUERY = gql`
 const Party = () => {
   const { party } = useParams<{ party: string }>();
   const partyId = party ? toNonHex(party) : '';
+  const filters = `filters[tx.submitter]=${partyId}`;
+  const { hasMoreTxs, loadTxs, error, txsData, loading } = useTxsData({
+    limit: 10,
+    filters,
+  });
 
   const {
     state: { data: partyData },
-  } = useFetch<BlockExplorerTransactions>(
-    `${DATA_SOURCES.blockExplorerUrl}/transactions?limit=1&filters[tx.submitter]=${partyId}`
+  } = useFetch<TendermintSearchTransactionResponse>(
+    `${DATA_SOURCES.tendermintUrl}/tx_search?query="tx.submitter='${partyId}'"`
```

Comment:
suggestion (non-blocking): composing queries manually feels quite hacky and it's borderline illegible, consider using some builder, like https://www.npmjs.com/package/query-string